microservice/package/tmp/Dockerfile

```diff
@@ -0,0 +1,2 @@
+from busybox
+CMD echo "this is a microservice"
```

Comment:
Do you really want to merge this `tmp` directory ?

Reply:
Whoops. This was another leftover from local testing, which should not have made it in the PR. This was added in error, sorry for that. The files have been removed with the last commit.